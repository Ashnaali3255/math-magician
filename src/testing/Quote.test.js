import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../Components/quote';

it('check if the Quote section renders properly', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
