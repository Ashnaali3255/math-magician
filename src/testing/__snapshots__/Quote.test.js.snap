// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check if the Quote section renders properly 1`] = `
<div
  className="quote-container"
>
  <p>
    Loading...
  </p>
</div>
`;
